tests/nodeos_high_transaction_test.py

```diff
@@ -285,7 +319,7 @@ def findTransInBlock(transId, transToBlock, node):
                     # delay and see if transfer is accepted now
                     Utils.Print("Transfer rejected, delay 1 second and see if it is then accepted")
                     time.sleep(1)
-                trans=node.transferFunds(fromAccount, toAccount, transferAmount, "transfer round %d" % (round), exitOnError=False, reportStatus=False, sign = True, dontSend = args.send_duplicates)
+                trans=transfer(node, contract, fromAccount, toAccount, transferAmount)
```

Comment:
If I understand it correctly, you are modifying the script only to use `token.kv` instead of having a configuration to choose between `eosio.token` and `token.kv`. Is that right?

Reply:
yes, but I went to the more generic use of "push action" with "transfer" than using the "transfer" command provided by cleos.